### ISSUE TITLE
Generate seeds using @stablelib/random

### ISSUE THREAD

**Issue by TimoGlastra** [hyperledger/aries-framework-javascript#957]
We could use `@stablelib/random` for this right? https://github.com/StableLib/stablelib/blob/d8b909d49245c040e5e4a59ff8c9b269fc1781ee/packages/random/random.ts#L52

_Originally posted by @blu3beri in https://github.com/hyperledger/aries-framework-javascript/pull/953#discussion_r925283552_

We need to check whether @stablelib/random works in react native through react-native-get-random-values

**Comment by blu3beri** [comment]
the bbs-signatures library from mattrglobal uses it for the randomness of the seed. 

https://github.com/mattrglobal/bbs-signatures/blob/3a55cffdda65f894e4e9a2bd89aa9ce483c611c0/src/js/wasm_module.js#L16

`@stablelib/random` detects platform and sets randomnesssource here: https://github.com/StableLib/stablelib/blob/d8b909d49245c040e5e4a59ff8c9b269fc1781ee/packages/random/source/system.ts#L8

- Node: https://github.com/StableLib/stablelib/blob/d8b909d49245c040e5e4a59ff8c9b269fc1781ee/packages/random/source/node.ts#L9
- Browser: https://github.com/StableLib/stablelib/blob/d8b909d49245c040e5e4a59ff8c9b269fc1781ee/packages/random/source/browser.ts#L8

I am not sure how `react-native-get-random-values` polyfills, but if its injected here we could make it work. Otherwise opening a PR adding React Native support would be trivial with `react-native-get-random-values`.

**Comment by TimoGlastra** [comment]
react-native-get-random-values polyfill the browser crypto object, so it should just work. The only thing where it could fail is if stablelib/random fails to detect the RN envrionment as a browser environment. I'll quickly test it/

**Comment by blu3beri** [comment]
Okay thats sad. I am not sure if `@stablelib` is big on RN support but we could add it.

**Comment by TimoGlastra** [comment]
Yeah we don't even need to add a dependency, just extend the browser check to also work in the case of RN


https://cs.github.com/StableLib/stablelib/blob/d8b909d49245c040e5e4a59ff8c9b269fc1781ee/packages/random/source/system.ts#L8

I think the issue is that is checks for `self.crypto`, and we need to check for just the `crypto` global

**Comment by blu3beri** [comment]
> I think the issue is that is checks for `self.crypto`, and we need to check for just the `crypto` global

Yeah that seems very likely. Could you test that in your test app? just patch the node module.

**Comment by TimoGlastra** [comment]
It does work 🎉, I just imported the react-native-get-random-values library at the wrong place so @stablelib/random got imported before. Adding it as the first thing in index.js fixed it and it seems to work out of the box.

So we can just make a PR and start using that as the seed generator
